docs/stencil-docs/installing-stencil-cli/troubleshooting-your-setup.mdx

```diff
@@ -6,6 +6,36 @@ For any unexpected behavior you encounter while developing your Stencil theme, w
 
 In some cases, the terminal will provide a verbose error message specifying where to look for problems. It has the potential to provide further insight on the issue. Diagnostic suggestions are listed on this page for error messages that may not be helpful in revealing the issue you're experiencing.
 
+<Callout type="warning">
+BigCommerce is currently targeting 11/1/2023 to sunset its node-sass fork in favor of the latest [sass/node-sass](https://github.com/sass/node-sass). To ensure that your storefront is up to date, use the latest node version (Node 18) in Stencil CLI and use the CLI command to resolve [incompatible SCSS directives](/stencil-docs/installing-stencil-cli/troubleshooting-your-setup#incompatible-scss-directives), which can cause issues with the styling of your storefront.
+</Callout>
+
+## Incompatible SCSS directives
+
+To ensure that your storefront is up to date, perform the following steps:
+
+1. Using the latest version of Stencil CLI, run the stencil bundle command to validate your theme's code. Note any errors in the console output.
+```shell copy
+stencil bundle
+```
+2. Update your files to ensure the correct formatting by running the following script. This step allows you to see changes before merging.
+```shell showLineNumbers copy
```

Comment:
The other code samples on this page don't show line numbers if they are only one line, so we should be consistent across the entire page